### DISCUSSION TITLE
Shelly Plus Uni

### DISCUSSION THREAD

**Discussion by soporte97** [arendst/Tasmota#21942] [Q&A]
Good afternoon, first of all, I would like to thank the great work you do for the community. I have been observing the compatible devices and templates, but I can't find anything for the Shelly Plus UNI although the device is equipped with an ESP32-U4WDH. I hope you can help me install TASMOTA or include it in the compatible list. I will attach the information I have. Thank you.

MCU
- CPU: | ESP32-U4WDH
- Flash: | 4 MB

![Shelly Plus UNI specs](https://github.com/user-attachments/assets/f11f8cf0-cb5b-4789-a54a-b7048fcd8369)
![Plus-Uni-addon-interface](https://github.com/user-attachments/assets/3420c12a-36ad-4d1f-b7a3-515138ee8817)
![Plus-Uni-internal-schematics](https://github.com/user-attachments/assets/10449749-aa14-4d6d-b288-1d72217369ad)

**Comment by sfromis** [comment]
Since the device is mentioned on https://github.com/tasmota/mgos32-to-tasmota32 it should just be the standard flashing procedure for OTA flashing. And that 7-pin connector is also the same as with other Shelly devices, meaning wired flashing as any other ESP32 when connecting there.
